Tasks/AppCenterDistributeV3/appcenterdistribute.ts

```diff
@@ -404,16 +404,16 @@ function expandSymbolsPaths(symbolsType: string, pattern: string, continueOnErro
 
     let symbolsPaths: string[] = [];
 
-    if (symbolsType === "Apple") {
+    if (symbolsType === "Apple" || symbolsType === "Breakpad") {
         // User can specifay a symbols path pattern that selects
         // multiple dSYM folder paths for Apple application.
-        let dsymPaths = utils.resolvePaths(pattern, continueOnError, packParentFolder);
+        let symPaths = utils.resolvePaths(pattern, continueOnError, packParentFolder);
 
         // Resolved paths can be null if continueIfSymbolsNotFound is true and the file/folder does not exist.
-        if (dsymPaths) {
-            dsymPaths.forEach(dsymFolder => {
-                if (dsymFolder) {
-                    let folderPath = utils.checkAndFixFilePath(dsymFolder, continueOnError);
+        if (symPaths) {
```

Comment:
Nitpick: This was named `dsymPaths` before since it only referenced dSYMs - we could name it (and according variables) `symbolPaths` now instead - also using `symbolFolder` instead of `symFolder` below, i.e.